app/src/main/java/se/inera/intyg/minaintyg/certificate/service/CertificateTextConverter.java

```diff
@@ -0,0 +1,49 @@
+package se.inera.intyg.minaintyg.certificate.service;
+
+import java.util.List;
+import java.util.function.Function;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.CertificateLink;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.CertificateText;
+import se.inera.intyg.minaintyg.util.html.HTMLTextFactory;
+
+@Service
+@RequiredArgsConstructor
+public class CertificateTextConverter {
+
+  public String convert(CertificateText text) {
```

Comment:
Consider calling this variable certificateText, since text might be missleading.

Reply:
Since the variable lives inside of CertificateTextConverter I think adding certificateText is redundant, similar to why I would avoid naming an id of a certificate certificateId instead of id if it lives inisde a certificate service. But I can update it.